transactions/transactor.go

```diff
@@ -68,25 +65,21 @@ func (t *Transactor) SetRPC(rpcClient *rpc.Client, timeout time.Duration) {
 }
 
 // SendTransaction is an implementation of eth_sendTransaction. It queues the tx to the sign queue.
-func (t *Transactor) SendTransaction(ctx context.Context, args SendTxArgs) (gethcommon.Hash, error) {
-	if ctx == nil {
-		ctx = context.Background()
-	}
-
-	completeFunc := func(acc *account.SelectedExtKey, password string, signArgs *sign.TxArgs) (sign.Response, error) {
-		t.mergeSignTxArgsOntoSendTxArgs(signArgs, &args)
-		hash, err := t.validateAndPropagate(acc, args)
-		return sign.Response(hash.Bytes()), err
-	}
+func (t *Transactor) SendTransaction(sendArgs SendTxArgs,
+	signArgs *sign.TxArgs, verifiedAccount *account.SelectedExtKey) sign.Result {
```

Comment:
Do I need the signArgs argument anymore or should we just leave gas and gasprice in the regular txargs?

Reply:
no idea, that wasn't me who introduced them :(